packages/angular/projects/dev/src/app/app.routing.ts

```diff
@@ -77,10 +77,10 @@ export const APP_ROUTES: Routes = [
     path: 'password',
     loadChildren: () => import('./password/password.demo.module').then(m => m.PasswordDemoModule),
   },
-  {
-    path: 'popovers',
-    loadChildren: () => import('./popovers/popovers.demo.module').then(m => m.PopoversDemoModule),
-  },
+  // {
```

Comment:
Can we add a comment to why this is disabled for now?

Reply:
Sure